packages/web-components/src/divider/divider.styles.ts

```diff
@@ -0,0 +1,21 @@
+import { css } from '@microsoft/fast-element';
+
+import {} from '../theme/design-tokens.js';
+
+export const styles = css`
+  :host {
+  }
+
+  :host([alignContent='start']) {
+    display: flex;
+    align-items: flex-start;
+  }
+  :host([alignContent='center']) {
+    display: flex;
+    align-items: center;
+  }
+  :host([alignContent='end']) {
+    display: flex;
+    align-items: end;
+  }
+`;
```

Comment:
Looks like a few styles missing here - in terms of attribute selectors, this works given the current syntax but be sure to update when we update to use valid attribute syntax.

Reply:
I'll update this once I have the component in working order.